test/api_auth.test.js

```diff
@@ -20,14 +31,15 @@ describe('Auth0 api-auth client grants', function () {
     client_secret: process.env.CIMPRESS_IO_CLIENT_SECRET
   };
 
-  it('Should make a successful request against the backing API', function(done) {
+  it('Should store the access token in the provided cache', function(done) {
```

Comment:
I think you should create this as a new test rather than a replacement of the existing test.